### PR TITLE
Fix and improvements for levels 4, 10, 11, 12, 13, 14, 23

### PR DESCRIPTION
**Pull request by viamodulo**

### CODE REVIEW

**Comment by viamodulo** [comment]
I'm not quite sure why the CI breaks after this :

13-Equalization-Room-9.27/13.25.specific-Gimlao.asm
  [13 instructions] [15 steps] [90% pass]

If I can do anything let me know.

**Comment by atesgoral** [comment]
@viamodulo Thanks for these! I'll take a look at CI soon.

File: solutions/13-Equalization-Room-9.27/14.25.viamodulo.asm [hunk 1]
```diff
@@ -0,0 +1,25 @@
+-- HUMAN RESOURCE MACHINE PROGRAM --
```
Comment (atesgoral) [inline solutions/13-Equalization-Room-9.27/14.25.viamodulo.asm]: I found the issue!

Due to the way I was mishandling path parsing errors, a clear error wasn't being reported. I fixed it on master: https://github.com/atesgoral/hrm-solutions/commit/6b880f5f9dfbb98530b71c97f696ca9d871ba295

`14.25.viamodulo.asm` should be `14.25-viamodulo.asm`. I'll merge your PR and fix it on master.

Comment (atesgoral) [reply solutions/13-Equalization-Room-9.27/14.25.viamodulo.asm]: Here's the proper error: https://travis-ci.org/atesgoral/hrm-solutions/builds/641128691#L358

Comment (atesgoral) [reply solutions/13-Equalization-Room-9.27/14.25.viamodulo.asm]: Fix: https://github.com/atesgoral/hrm-solutions/commit/d878e4aac6aa726dd7e324284573f548c5864b90

**Pull request merged**